### DISCUSSION TITLE
Can `delta` show `patch` files?

### DISCUSSION THREAD

**Discussion by NightMachinery** [dandavison/delta#725] [Q&A] [answered]
Can `delta` show `patch` files? (produced by `git format-patch ...`)

**Comment by NightMachinery** [comment, marked as answer]
`cat file.patch | delta` works.

**Comment by Kr1ss-XD** [comment]
Yes, absolutely. You can provide any kind of unified diff output to its stdin, e.g.

```sh
delta <file.patch
```

or

```sh
git format-patch --stdout HEAD^ | delta
```

If you have something in your git config like : 

```gitconfig
[pager]
	format-patch = delta
```

then you can also just

```sh
git format-patch --stdout HEAD^
```

without the explicit pipe.
